hepcrawl/spiders/aps_spider.py

```diff
@@ -124,10 +123,12 @@ def _parse_jats(self, response):
 
         file_name = self._file_name_from_url(response.url)
         parser.attach_fulltext_document(file_name, response.url)
```

Comment:
I don't know if it's happening, but the attached fulltext should be hidden

Reply:
Same here :) https://github.com/inspirehep/hepcrawl/blob/52db9383d077c24b726137128428abff0a20af38/hepcrawl/parsers/jats.py#L588-L589